tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -22,6 +22,18 @@
 from openvino.tools.benchmark.utils.statistics_report import StatisticsReport, JsonStatisticsReport, CsvStatisticsReport, \
     averageCntReport, detailedCntReport
 
+import psutil
+
+def get_peak_memory_usage():
+    process = psutil.Process()
+    mem_info = process.memory_info()
+    mem_multiplier = 1024
+    
+    if hasattr(mem_info, 'peak_wset'):
+        return int(mem_info.peak_wset / mem_multiplier)
+    else:
+        raise AttributeError("Can't get system memory values.")
```

Comment:
`peak_wset` is known to be missing. This error is going to be triggered. https://psutil.readthedocs.io/en/latest/#psutil.Process.memory_info You need another way to handle this: find a more general approach or don't log anything if `peak_wset` is missing

Reply:
Refactored. Please check

Reply:
C++ Linux fails if something is wrong with `/proc/self/status` but Python Linux skips printing in that case.

Since you always have a way to handle VmPeak now in python
> You need another way to handle this: find a more general approach or don't log anything if peak_wset is missing

becomes not applicable. You should always print

Reply:
You need to specify what you don't get

Reply:
Does it look good now for your?